packages/matchbox/src/components/Grid/styles.js

```diff
@@ -0,0 +1,103 @@
+import { tokens } from '@sparkpost/design-tokens';
+
+const gutterWidth = tokens.spacing_400;
+export const breakpoints = ['xs', 'sm', 'md', 'lg', 'xl'];
+const alignmentStyleMap = {
+  start: {
+    justifyContent: 'flex-start',
+    textAlign: 'start',
+  },
+  center: {
+    justifyContent: 'center',
+    textAlign: 'center',
+  },
+  end: {
+    justifyContent: 'flex-end',
+    textAlign: 'end',
+  },
+  top: {
+    alignItems: 'flex-start',
+  },
+  middle: {
+    alignItems: 'center',
+  },
+  bottom: {
+    alignItems: 'flex-end',
+  },
+  around: {
+    justifyContent: 'space-around',
+  },
+  between: {
+    justifyContent: 'space-between',
+  },
+};
+
+const wrapStyleWithMediaQuery = (breakpoint, style) => {
```

Comment:
This is a bit unfortunate :( If we could change the API styled-system has helpers to handle all of this pretty easily